.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can we fix tab spaces here? We use 2, and in my case when I save this file, prettier will update format